### PR TITLE
Update java/rocksjni.pom

### PR DESCRIPTION
**Pull request by siying**

### CODE REVIEW

File: java/rocksjni.pom [hunk 1]
```diff
@@ -19,6 +19,11 @@
             <url>http://www.apache.org/licenses/LICENSE-2.0.html</url>
             <distribution>repo</distribution>
         </license>
+        <license>
+            <name>GNU General Public License, version 2</name>
+            <url>http://opensource.org/licenses/gpl-2.0</url>
```
Comment (sagar0) [inline java/rocksjni.pom]: Wondering if the URL should actually link to https://www.gnu.org/licenses/old-licenses/gpl-2.0.en.html ?

Comment (siying) [inline java/rocksjni.pom]: The one we are using is https://www.gnu.org/licenses/gpl-2.0.txt
Maybe I should put that one.

Comment (siying) [inline java/rocksjni.pom]: Or https://www.gnu.org/licenses/gpl-2.0.html

Comment (sagar0) [inline java/rocksjni.pom]: yeah, google search shows those links from gnu.org to be more authentic and lot more used. So, either one of those should be good.

**Comment by adamretter** [comment]
@siying looks good to me.

**Pull request closed without merging**